pilot/pkg/autoregistration/controller.go

```diff
@@ -23,67 +23,27 @@ import (
 	"time"
```

Comment:
nit: this is no longer the "autoregistration" controller, it's the WorkloadEntry controller

Reply:
yes, I'm planning to rename the package into `workloadentry`.

I left such renaming for a separate PR to improve readability of this one.